integration/test_resources/help/cloud-account_iac-generate_azure

```diff
@@ -0,0 +1,53 @@
+Use this command to generate Terraform code for deploying Lacework into new Azure environment.
+
+By default, this command will function interactively, prompting for the required information to setup the new cloud account. In interactive mode, this command will:
+		
+* Prompt for the required information to setup the integration
+* Generate new Terraform code using the inputs
+* Optionally, run the generated Terraform code:
+  * If Terraform is already installed, the version will be confirmed suitable for use
+	* If Terraform is not installed, or the version installed is not suitable, a new version will be installed into a temporary location
+	* Once Terraform is detected or installed, Terraform plan will be executed
+	* The command will prompt with the outcome of the plan and allow to view more details or continue with Terraform apply
+	* If confirmed, Terraform apply will be run, completing the setup of the cloud account
+
+Usage:
+  lacework cloud-account iac-generate azure [flags]
+
+Aliases:
+  azure, az
+
+Flags:
+      --activity_log                           enable active log integration
+      --activity_log_integration_name string   specify a custom activity log integration name
+      --ad_create                              create new active directory integration (default true)
+      --ad_id string                           existing active directory application id
+      --ad_pass string                         existing active directory application password
+      --ad_pid string                          existing active directory application service principle id
+      --all_subscriptions subscription ids     grant read access to ALL subscriptions within Tenant (overrides subscription ids)
+      --config                                 enable configuration integration
+      --config_name string                     specify a custom configuration integration name
+      --existing_storage                       use existing storage account
+  -h, --help                                   help for azure
+      --location string                        specify azure region where storage account logging resides
+      --management_group                       management group level integration
+      --management_group_id string             specify management group id. Required if mgmt_group provided
+      --output string                          location to write generated content
+      --storage_account_name string            specify storage account name
+      --strg_res_group string                  specify storage resource group
+      --sub_ids strings                        list of subscriptions to grant read access; format is id1,id2,id3
```

Comment:
Just noticing these aren't in line with the other flags.
Can we update to
```
--storage_resource_group
--subscription_ids
```

Reply:
Done